scripts/presets/travis/standalone/ecdsa-basic-tls-ocsp-disableStapling.json

```diff
@@ -0,0 +1,19 @@
+{
+   "id" : "standalonenoauthssl",
+   "name": "mongod",
+   "procParams": {
+      "ipv6": true,
+      "bind_ip": "127.0.0.1,::1",
+      "logappend": true,
+      "journal": true,
+      "port": 27017,
+      "setParameter": {"failpoint.disableStapling":"{\"mode\":\"alwaysOn\"}}"}
+   },
+   "sslParams": {
+      "sslOnNormalPorts": true,
+      "sslPEMKeyFile": "$TRAVIS_BUILD_DIR/.evergreen/ocsp/ecdsa/server.pem",
+      "sslCAFile": "$TRAVIS_BUILD_DIR/.evergreen/ocsp/ecdsa/ca.pem",
```

Comment:
Working on PHPC-1626 would also take care of this, since we'd no longer be using our own orchestration configs after that (at least for most configs).